### PR TITLE
Added capsule shape

### PR DESCRIPTION
**Pull request by VladimirIvan**
Same usage and features as the cylinder.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name = "GeometryTypes"
-uuid = "4d00f742-c7ba-57c2-abde-4428a4b178cb"
+uuid = "01225300-7f3c-11ea-2c6e-fdcb135c3b29"
```
Comment (sjkelly) [inline Project.toml:2]: The UUID shouldn't change.

**Comment by SimonDanisch** [comment]
Btw, you need to ping me when it's ready to merge!

**Comment by VladimirIvan** [comment]
@sjkelly Thank you for the review.
@SimonDanisch All changes done.

**Comment by SimonDanisch** [comment]
Btw, would you mind opening the same PR over at GeometryBasics as well?
GeometryTypes will be replaced by GeometryBasics!

**Comment by SimonDanisch** [comment]
closed in favor of #205

**Pull request closed without merging**